extern/CMakeLists.txt

```diff
@@ -109,21 +109,18 @@ else()
 endif()
 
 if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
-    find_package(GLEW REQUIRED)
-    target_link_libraries(ImGui PUBLIC opengl32 GLEW::GLEW)
```

Comment:
Because it's not required anymore.
`SDL2_opengl.h` has the functions previously provided by the GLEW headers.

Maybe I should rename the PR to remove the GNU/Linux part?